tests/test_optimal_control/test_ocp.py

```diff
@@ -30,7 +30,7 @@ def test_set_bounds(self):
 
         def _set_and_assert(group: str, name: str, bound_vals: Tuple[Any]):
             new_bounds = (StageVarBoundConfig(group, name, bound_vals),)
-            self.ocp.set_bounds(new_bounds)
+            self.ocp.update_bounds(new_bounds)
```

Comment:
update tests to reflect the new way of updating bounds? setting default and update, no longer stateless

Reply:
done